openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1389,3 +1397,109 @@ def verify_setup(self, prefix, timeout=300, wait_step=10):
                 h_vol_list, g_vol_list, pv_vols))
 
         self.verify_pods_are_running()
+
+    def create_heketi_volumes_in_batch(
+            self, heketi_client_node, heketi_server_url, vol_count, size=1,
+            batch_amount=8, timeout=600, interval=5, **kwargs):
```

Comment:
why is the default timeout value is high, we can have default limit any changes has to come from testcase to increase the limit

Reply:
This was due to incorrect rebase